content/learn/vp_openshift_framework.adoc

```diff
@@ -0,0 +1,304 @@
+
+== Applications and `values-` files
+
+Helm uses `values.yaml` files to pass variables into charts. Variables in the `values.yaml` file can be overridden in the following ways:
+
+. By a `values.yaml` file in the parent directory
+. By a values file passed into the `helm <install/upgrade>` command using `-f`
+. By specifying an override individual value in the the `helm` command with `--set`
+
+For more information on values files and their usage see the https://helm.sh/docs/chart_template_guide/values_files/[values files section] of the Helm documentation.
+
+This section is meant as an introduction to the `values-` files that the framework uses to override values in the chart templates. In the Getting Started pages there will be more specific usage details.
+
+=== There are three types of `value-` files.
+
+. *`values-global.yaml`*:
+This is used to set variables for helm charts across the pattern. It contains the name of the pattern and sets some other variables for artifacts like, image registry, Git repositories, GitOps syncPolicy etc.
+. *`values-<site>.yaml`*:
+Each specific site requires information regarding what applications and subscriptions are required for that site. This file contains a list of namespaces, applications, subscriptions, the operator versions etc. for that site.
+. *`values-secret.yaml.template`*:
+All patterns require some secrets for artifacts included in the pattern. For example credentials for GitHub, AWS, or Quay.io. The framework provides a safe way to load those secrets into a vault for consumption by the pattern. This template file can be copied to your home directory, the secret values applied, and the validated pattern will go look for `values-secrets.yaml` in your home directory. *Do not leave a `values-secrets.yaml` file in your cloned git directory or it may end up in your (often public) Git repository, like GitHub.*
+
+=== Values files can have some overrides.
+
+. Version overrides can be used to set specific values for OCP versions. For example *`values-hub-4.16.yaml`* allows you to tweak a specific value for OCP 4.16 on the Hub cluster.
+. Version overrides can be used to set specific values for specific cloud providers. For example *`values-AWS.yaml`* would allow you to tweak a specific value for all cluster groups deployed on AWS.
+
+=== Other combination examples include:
+
+. *`values-hub-Azure.yaml`* only apply this Azure tweak on the hub cluster.
+. *`values-4.16.yaml`* apply these OCP 4.16 tweaks to all cluster groups in this pattern.
+
+Current supported cloud providers include *`AWS`*, *`Azure`*, *`GCP`*, and *Nutanix*.
+
+== Applications & subscriptions
+
```

Comment:
Should applications & subscriptions have some text here or should we change the indentation?

Reply:
Removed as seems out of place here